packages/office-ui-fabric-react/src/components/FocusTrapZone/examples/FocusTrapZone.Box.Example.tsx

```diff
@@ -44,11 +44,9 @@ export default class BoxExample extends React.Component<React.HTMLAttributes<HTM
     const { isChecked } = this.state;
 
     return (
-      <div className="ms-FocusTrapZoneBoxExample">
+      <div className={contentClass}>
```

Comment:
Is there a reason not to use a `Stack` here? Seems to be a need for a container with styling, which sounds like a `Stack` case to me. Since this is an example I'd rather steer people towards Stack than div with `mergeStyles` usage. (This comment applies to multiple examples in this PR.)

Reply:
Good point. I made them all use Stacks (and look nicer). While doing that and looking at the examples, I realized that the toggles in most of the examples were totally redundant with the buttons, so I took them out. Let me know if you disagree with that.

Reply:
The only concern I have with removing the Toggle is that it's less obvious that there is a FocusTrapZone active preventing interaction with the rest of the page...

Reply:
Hmm interesting. Maybe I could remove the buttons instead? It just seems wrong to have multiple ways of entering/exiting the trap zone...

Reply:
That seems fair. I think the Toggles make it a little more clear when an FTZ is active and preventing you from interacting with the rest of the component page:
![image](https://user-images.githubusercontent.com/26070760/55261538-7b298980-5228-11e9-8af3-4458a4e9233f.png)